BrewTunes/platforms/android/assets/www/js/app.js

```diff
@@ -126,7 +126,7 @@ $(document).on('ready', function() {
     $(document).on('click', '#submit', function() {
         reset(); // for subsequent searches
         input = $('#input-beer').val().trim();
-        if (input.match(/^[\w\-\s]+$/)) {
+        if (input.match(/^[\w\-\s\']+$/)) {
```

Comment:
correct. this is just because that is what changed since the previous build. we changed it in the app.js in the www folder, but not in the android specific www folder, which gets updated on the build, which i did after it was previously added.